libyul/backends/wasm/WasmDialect.h

```diff
@@ -45,6 +45,8 @@ struct WasmDialect: public Dialect
 	WasmDialect();
 
 	BuiltinFunction const* builtin(YulString _name) const override;
+	BuiltinFunction const* discardFunction() const override { return builtin("drop"_yulstring); }
+	BuiltinFunction const* equalityFunction() const override { return builtin("i64.eq"_yulstring); }
```

Comment:
How can we use a 64-bit comparison here where we have a 256-bit comparison for EVM? It's used for ``switch``->``if`` below - will WASM ``switch``es be 64-bit and not simulate 256 bits?

Reply:
This dialect only knows 64 bit types (as per the comment on the class).